templates/deployment.yaml

```diff
@@ -43,6 +43,15 @@ spec:
                 secretKeyRef:
                   key: runnerToken
                   name: {{ .Values.configSecret.name }}
+            - name: "CIRCLECI_RUNNER_API_AUTH_TOKEN"
```

Comment:
Shouldn't we also remove `CIRCLECI_API_TOKEN`?

Reply:
Not now. Our terraform-aws-eks module installs shared-fleet-infra's main branch to all clusters. So once we merge these changes, runners that are still in launch-agent would fail.

This clean up should happen after migration is complete . Also, we should update livenessProbe as well